include/ola/thread/Thread.h

```diff
@@ -33,18 +33,6 @@
 
 #include <string>
 
-#if defined(_WIN32) && defined(__GNUC__)
```

Comment:
Yeah @peternewman did this in this commit:

https://github.com/OpenLightingProject/ola/commit/c30041c1095246ae578c66bc53f5a8f464052e32

I don't think we can remove.